src/presenters/admin/indexation-warning-presenter.php

```diff
@@ -34,10 +44,12 @@ class Indexation_Warning_Presenter extends Abstract_Presenter {
 	 * Indexation_Warning_Presenter constructor.
 	 *
 	 * @param int            $total_unindexed The number of objects that needs to be indexed.
+	 * @param bool           $action_is_link  Whether the action is a link or a button.
 	 * @param Options_Helper $options_helper  The options helper.
 	 */
-	public function __construct( $total_unindexed, Options_Helper $options_helper ) {
+	public function __construct( $total_unindexed, $action_is_link, Options_Helper $options_helper ) {
```

Comment:
You have to add the `action_is_link` as last argument, because this is a public function that has been released already.

Reply:
Good call!